src/js/components/Box/Box.js

```diff
@@ -164,6 +165,10 @@ const Box = forwardRef(
   },
 );
 
+const isChildNull = (children) => {
+  return !Boolean(ReactDOMServer.renderToStaticMarkup(children));
```

Comment:
Hmm... this feels like it could be rather heavy, rendering all children to string all of the time, especially if the children are complex.

I'm wondering if instead of doing it this way, we should explore a different means of adding the gap other than inserting extra elements into the DOM. For instance, using styled-components to include a child selector that adds a margin to all children except the last one?

Reply:
yes, I’m also worried about the performance implications, but haven’t really tested that aspect.

I’m happy to try your suggested approach.